### ISSUE TITLE
Mastodon link verification

### ISSUE THREAD

**Issue by JeremyStarTM** [louislam/uptime-kuma#3207]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature, Other

### 🔖 Feature description

The ability to insert rel="me" links so Mastodon can show status pages as verified.

### ✔️ Solution

A textbox where one or more usernames can be inserted

### ❓ Alternatives

Inserting <link> tags into the footer or description should be possible too.

**Comment by CommanderStorm** [comment]
Out of curiosity: Why would adding an [uptime monitor to your profile](https://docs.joinmastodon.org/user/profile/#verification) be something you want?

I think that mastodon is quite a niche product (1M users), probably the ability to add extra meta tags would be something more applicable, right?

**Comment by JeremyStarTM** [comment]
It's a way to show that a user is the person it claims to be. It's like the blue checkmark but better and for everyone.

**Comment by CommanderStorm** [comment]
Let me rephrase the questions:
Out of curiosity: Why do you want to add the link to an uptime monitor (instead of a personal website) to your profile?)

**Comment by JeremyStarTM** [comment]
Just because. I want to link status.foo.bar, not foo.bar/status.

**Comment by kmcbride3** [comment]
That doesn't really answer the question - what is the benefit of adding a feature to enable verification of your Uptime Kuma status page on Mastodon?

We're asking what the importance/relevance of having this capability within Uptime Kuma is. It doesn't seem like social media verification is an obvious fit for Uptime Kuma's primary purpose, which is to monitor services and send notifications or alerts based on the health status of said services.

Is it that you want (or need) the ability to verify yourself (or your business) on Mastodon, and you already have a publicly accessible Uptime Kuma instance running, so you're looking for an easy way to add the rel="me" link Mastodon requires to the status page to facilitate this? You're likely better off spinning up a simple webpage for that purpose instead.

**Comment by hmhackmaster** [comment]
As a Mastodon admin myself I do see a value to allow instance 'support' accounts to be able to have the instance's official status page listed in the account's links (with the verified check).

Unfortunately, the only available text field on the uses Markdown for formatting and I am unaware of any way to add the `rel="me"` relationship attribute in Markdown, so the only trivial(ish) way to make this work would be to add the `rel="me"` attribute to all links in the "Description" field. 

Genuinely not sure if that would make sense.

**Comment by CommanderStorm** [comment]
@JeremyStarTM I think this issue can be merged into https://github.com/louislam/uptime-kuma/issues/3115 which might get resolved by https://github.com/louislam/uptime-kuma/pull/3693

**Comment by CommanderStorm** [comment]
@JeremyStarTM 
We are consolidating duplicate issues a bit to make issue management easier.
I think, we should track this issue in https://github.com/louislam/uptime-kuma/issues/2818 as there is no functional difference that I can see ^^
=> I am going to close this as a duplicate (despite being the older issue)
